lib/modules/manager/bazelisk/index.ts

```diff
@@ -0,0 +1,10 @@
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.bazelversion$'],
```

Comment:
```suggestion
  fileMatch: ['(^|\\/)\\.bazelversion$'],
```
should properly escaped

Reply:
I copied this from elsewhere such as `.nvmrc` support. If you search for `'(^|/)\\.` this exact snippet is used in a few configs, while the one you're suggesting is used in 1.

I'm not sure which is correct? The `/` doesn't have to be escaped in a regex (unlike the `.`) so I wouldn't think it's needed.

Reply:
i know it works without escape because of relaxed regex engine. so please escape

Reply:
Is that because `/` is a special character in a non-relaxed engine? 👍